test/1.2/errors/T_WBIGUNROLL.dml

```diff
@@ -0,0 +1,80 @@
+/*
+  © 2023 Intel Corporation
+  SPDX-License-Identifier: MPL-2.0
+*/
+dml 1.2;
+
+device test;
+
+/// COMPILE-ONLY
+/// NO-CC
+
+data int zero = 0;
+
+parameter zeroes_50 = [$zero, $zero, $zero, $zero, $zero, $zero, $zero, $zero,
+                       $zero, $zero, $zero, $zero, $zero, $zero, $zero, $zero,
+                       $zero, $zero, $zero, $zero, $zero, $zero, $zero, $zero,
+                       $zero, $zero, $zero, $zero, $zero, $zero, $zero, $zero,
+                       $zero, $zero, $zero, $zero, $zero, $zero, $zero, $zero,
+                       $zero, $zero, $zero, $zero, $zero, $zero, $zero, $zero,
+                       $zero, $zero];
```

Comment:
for readability, consider
```
param zeroes_10 = [$zero, $zero, $zero, $zero, $zero, $zero, $zero, $zero, $zero, $zero]
param zeroes_50 = $zeroes_10 + $zeroes_10 + $zeroes_10 + $zeroes_10 + $zeroes_10;
..
assert zeroes_50.len == 50;
```

Reply:
... you can concatenate compile-time lists?

Reply:
No you can't.